src/gui/sdlmain.cpp

```diff
@@ -4627,6 +4627,18 @@ int sdl_main(int argc, char *argv[])
 
 	int rcode = 0; // assume good until proven otherwise
 	try {
+		std::string working_dir;
+		constexpr bool remove_arg = true;
+		if (control->cmdline->FindString("--working-dir", working_dir, remove_arg) ||
+		    control->cmdline->FindString("-working-dir", working_dir, remove_arg)) {
+			std::error_code ec;
+			std::filesystem::current_path(working_dir, ec);
+			if (ec) {
+				LOG_WARNING("Cannot set working directory to %s",
+				            working_dir.c_str());
```

Comment:
This is so critical that it might warrant `LOG_ERR(..)` level (where as WARNING's are usually recoverable with some aspect being degraded or unavailable).

Reply:
Yeah I was vacillating between error and warning... Probably error is better, you're right. Changing.